easybuild/easyblocks/a/allinea.py

```diff
@@ -81,11 +82,34 @@ def install_step(self):
         # copy templates
         templ_path = os.path.join(self.installdir, 'templates')
         for templ in self.cfg['templates']:
+	    path = self.obtain_file(templ, extension='qtf')
+	    if path:
+		self.log.debug('Template file %s found' % path)
+	    else:
+		raise EasyBuildError('No template file named %s found', templ)
+
             try:
-                shutil.copy2(templ, templ_path)
+                shutil.copy(path, templ_path)
```

Comment:
If you mean the change from copy2 to copy, the installed file should have the same permissions as the rest of the installation, not whatever the template file had before the copy.